qa/tasks/scripts/install_ceph_salt.sh

```diff
@@ -0,0 +1,21 @@
+set -ex
+
+REPO=$1
+BRANCH=$2
+if [ -n "$REPO" ]
+then
+	if [ ! -n "$BRANCH" ]
+	then
+		BRANCH="master"
+	fi
+	cd /root
```

Comment:
Why do we need to go to `/root`?

Reply:
I had just copied the code from `sesdev` but I don't see any particular reason to clone the repo in `/root` dir so I removed it and testing it to be sure